packages/story-editor/src/output/page.js

```diff
@@ -85,11 +87,23 @@ function OutputPage({ page, autoAdvance = true, defaultPageDuration = 7 }) {
     )
     .map(({ id: videoId }) => `el-${videoId}-captions`);
 
+  const hasBackgroundAudioWithTracks =
+    backgroundAudio?.src && backgroundAudioTracks?.length > 0;
+
+  if (hasBackgroundAudioWithTracks) {
+    videoCaptions.push(`el-${backgroundAudio.id}-captions`);
```

Comment:
This needs to be the same ID that you pass to `HiddenAudio`, which is the page ID. Not `backgroundAudio.id`

```suggestion
    videoCaptions.push(`el-${id}-captions`);
```